src/Controls/src/Core/Platform/Windows/CollectionView/ScrollHelpers.cs

```diff
@@ -30,6 +30,29 @@ static UWPPoint AdjustToMakeVisible(UWPPoint point, UWPSize itemSize, ScrollView
 			return AdjustToMakeVisibleHorizontal(point, itemSize, scrollViewer);
 		}
 
+		static UWPPoint AdjustToStart(UWPPoint point, ScrollViewer scrollViewer, double height)
+		{
+			if (IsVertical(scrollViewer))
```

Comment:
```
return IsVertical(scrollViewer) 
        ? AdjustToStartVertical(point, height)
        : AdjustToStartHorizontal(point, height);
```

Reply:
@jsuarezruiz  ,I have addressed the return changes. Let me know if anything else needs adjustment.